schema/release-schema.json

```
@@ -107,6 +107,16 @@
       "description": "The details of related processes: for example, if this process follows on from one or more other processes, represented under a separate open contracting identifier (ocid). This is commonly used to relate mini-competitions to their parent frameworks or individual tenders to a broader planning process.",
       "title": "Related processes",
       "type": "array"
+    },
+    "links": {
+      "title": "Links",
+      "description": "Links to related resources. In a release, links relate to the individual release. For example, a link to a canonical version of the release in another OCDS publication. In a compiled release, links relate to the entire contracting process. For example, a link to a non-OCDS publication that represents the entire contracting process.",
```

Fix sentence fragments:

```suggestion
      "description": "Links to related resources. In a release, links relate to the individual release: for example, a link to a canonical version of the release in another OCDS publication. In a compiled release, links relate to the entire contracting process: for example, a link to a resource in a non-OCDS publication that represents the entire contracting process.",
```